pkg/curatedpackages/bundlesource_test.go

```diff
@@ -0,0 +1,91 @@
+package curatedpackages_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Can you use `gomega` for this?

Reply:
Not saying to not use gomega, just throwing out there, that the packages team has blessed the use of testify for our code. Obviously, this is your sandbox and we should play by your rules. :D